test/src/graph/nodes/Ros2PublishRadarScanNodeTest.cpp

```diff
@@ -0,0 +1,73 @@
+#include <helpers/commonHelpers.hpp>
+#include <helpers/sceneHelpers.hpp>
+#include <helpers/graphHelpers.hpp>
+#include <helpers/testPointCloud.hpp>
+
+#include <graph/NodesRos2.hpp>
+
+#include <rclcpp/rclcpp.hpp>
+
+// TODO: replace with API calls
+#include <api/apiCommon.hpp>
```

Comment:
```suggestion
#include <rgl/api/extensions/ros2.h>

#include <rclcpp/rclcpp.hpp>
#include <radar_msgs/msg/radar_scan.hpp>
```
Seems to be already resolved.